src/bin/facilitator-start.ts

```diff
@@ -20,8 +20,8 @@ process.on('SIGINT', terminationHandler);
 process.on('SIGTERM', terminationHandler);
 
 facilitatorCmd
-  .option('-mc, --mosaic-config <mosaicConfig>', 'path to mosaic configuration')
-  .option('-fc, --facilitator-config <facilitatorConfig>', 'path to facilitator configuration')
+  .option('-m, --mosaic-config <mosaicConfig>', 'path to mosaic configuration')
+  .option('-f, --facilitator-config <facilitatorConfig>', 'path to facilitator configuration')
```

Comment:
use `-c`.

Reply:
Using `-t` as `-c` is used in facilitator-init for chain id.